### PR TITLE
[IMP] surveys: rewrite of create surveys doc for 17

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3816462&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

This is, essentially, a complete rewrite of the current create.rst doc to include updated screenshots and more detailed steps/elements to paint a more "complete" picture for the reader.

This will solely focus on the survey form and its corresponding tabs. There will be another doc dedicated to question creation to follow this one. And, once that question-specific doc is published, I'll be sure to add a corresponding link to it in *this* doc.

I'm working off the current 'create.rst' file so as to not disrupt any potential links to this doc, which may be located elsewhere throughout the entire documentation.

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Alrighty, @Felicious -- this one is ready for Peer Review from you whenever you get a chance. Thanks!

File: content/applications/marketing/surveys/create.rst [hunk 1]
```diff
@@ -1,224 +1,334 @@
...
+
+- :guilabel:`Share`: click to reveal a :guilabel:`Share a Survey` pop-up form that can be used to
+  invite potential participants to the survey - complete with a :guilabel:`Survey Link` that can be
+  copied and sent to potential participants, and a :guilabel:`Send by Email` toggle switch.
+
+   .. image:: create/share-survey-popup.png
+      :align: center
+      :alt: The 'Share a Survey' pop-up window that appears in the Odoo Surveys application.
+
+  When the :guilabel:`Send by Email` toggle is in the 'on' position, indicated by a green switch,
```
Comment (Felicious) [inline content/applications/marketing/surveys/create.rst:289]: Attempt at brevity, feel free to ignore!
```suggestion
  When the :guilabel:`Send by Email` toggle is 'on' (green switch),
```

File: content/applications/marketing/surveys/create.rst [hunk 2]
```diff
@@ -1,224 +1,334 @@
...
-want to be listed in the Odoo database). Finally, click :guilabel:`Send`.
+- :guilabel:`Close`: clicking this button closes the survey (i.e. archives it), which is represented
+  by a red :guilabel:`Archived` banner across the top-right corner of the survey form.
 
-As answers are received, check them by clicking the :guilabel:`Answers` smart button on the survey
-template form, or the :guilabel:`See Results` button in the upper left corner. To end the survey,
-click the :guilabel:`Close` button on the survey template form.
+  When this button is clicked, and the survey is closed, a single button appears in the upper-right
+  corner of the survey form, titled: :guilabel:`Reopen`. When :guilabel:`Reopen` is clicked the
+  survey is reopened (i.e. unarchived), and the :guilabel:`Archived` banner is removed from the
```
Comment (Felicious) [inline content/applications/marketing/surveys/create.rst:333]: I don't believe unarchived is a real word, even though it totally makes sense in our corporate setting
```suggestion
  survey is reopened, and the :guilabel:`Archived` banner is removed from the
```

**Review by Felicious — APPROVED** [review]
Hello @ksc-odoo ! Great job on this extended overhaul of this doc! 

I just had some minor rewording suggestions to make the writing a little more concise in some places, but after you address those comments, feel free to move this onto the next stage of review!!

**Comment by ksc-odoo** [comment]
Thanks for the always-helpful feedback, @Felicious -- made all your suggested edits. This one is ready for Tech Review from @samueljlieber -- whenever you get a chance 👍

File: content/applications/marketing/surveys/create.rst [hunk 3]
```diff
@@ -1,224 +1,331 @@
...
-Clicking :guilabel:`Add a question` opens the :guilabel:`Create Sections and Questions` pop-up to
-create and customize the survey question.
+If the :guilabel:`Randomized per Section` option is enabled in the :guilabel:`Options` tab of the
+survey form, a column titled, :guilabel:`# Questions Randomly Picked` appears in the
+:guilabel:`Questions` tab.
 
-.. image:: create/survey-question-pop-up.png
+Indicate whether questions require a mandatory answer by clicking the :guilabel:`(slider)` icon to
+the far-right of the column titles. Then, select :guilabel:`Mandatory Answer` from the drop-down
+menu to reveal the :guilabel:`Mandatory Answer` column in the :guilabel:`Questions` tab.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:64]: Icon name 
```suggestion
Indicate whether questions require a mandatory answer by clicking the :guilabel:`(optional columns)`
icon to the far-right of the column titles. Then, select :guilabel:`Mandatory Answer` from the
drop-down menu to reveal the :guilabel:`Mandatory Answer` column in the :guilabel:`Questions` tab.
```

File: content/applications/marketing/surveys/create.rst [hunk 4]
```diff
@@ -1,224 +1,331 @@
...
-Create questions
-----------------
+Add a question
+~~~~~~~~~~~~~~
 
-In the :guilabel:`Create Sections and Questions` pop-up, type the question in the
-:guilabel:`Question` field. Then, choose the :guilabel:`Question Type`. A preview of how the
-question type looks is shown in the preview window.
+To add a question to a survey, click :guilabel:`Add a question` in the :guilabel:`Questions` tab,
+and proceed to fill out the :guilabel:`Create Sections and Questions` pop-up window that appears.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:74]: This is a modal UI element
```suggestion
and proceed to fill out the :guilabel:`Create Sections and Questions` modal that appears.
```

File: content/applications/marketing/surveys/create.rst [hunk 5]
```diff
@@ -1,224 +1,331 @@
...
 
-Choose from the following :guilabel:`Question Types`:
+To learn how to create and customize questions, refer to our documentation on that.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:76]: Let's link the documentation we are referencing :)
```suggestion
To learn how to create and customize questions, refer to the :doc:`create questions <time_random>`
documentation.
```

File: content/applications/marketing/surveys/create.rst [hunk 6]
```diff
@@ -1,224 +1,331 @@
...
-- :guilabel:`Numerical Value`
-- :guilabel:`Date`
-- :guilabel:`Datetime`
-- :guilabel:`Multiple choice: only one answer`
-- :guilabel:`Multiple choice: multiple answers allowed`
-- :guilabel:`Matrix`
+.. important::
+   There **must** be a survey title entered in order for the :guilabel:`Create Sections and
+   Questions` pop-up window to appear. If no title is entered ofr the survey, an error pop-up
+   message appears in the upper-right corner, instructing the user to enter a survey title.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:81]:
```suggestion
   Questions` modal to appear. If no title is entered for the survey, an error pop-up message
   appears in the upper-right corner, instructing the user to enter a survey title.
```

File: content/applications/marketing/surveys/create.rst [hunk 7]
```diff
@@ -1,224 +1,331 @@
...
-   the same, regardless of what question is chosen.
+Add a section
+~~~~~~~~~~~~~
 
-Create sections and questions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A *section* divides the survey into organized parts, in order to visually group similar questions
+together. To make a section, click :guilabel:`Add a section` at the bottom of the
+:guilabel:`Questions` tab, proceed to type in a desired name for the section, then either press
+'Enter' or click away.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:89]:
```suggestion
:kbd:`Enter` or click away.
```
or
```suggestion
`Enter` or click away.
```

File: content/applications/marketing/surveys/create.rst [hunk 8]
```diff
@@ -1,224 +1,331 @@
...
-- :guilabel:`Number of columns`: select how many columns are displayed.
-- :guilabel:`Images on answers`: allow images on the answer options.
-- :guilabel:`Conditional Display`: determine if the question is displayed based on the
-  participant's answer to a previous question.
-- :guilabel:`Show Comments Field`: allow the participant to type a comment in a text box.
-- :guilabel:`Question Time Limit`: for live session surveys, set a time limit for the question.
+This indicates that `1` question from that section will be picked at random for each participant
+taking the survey, bypassing every other, non-chosen question from that section. To change that
+number, select that figure, and type in the desired number in its place. Then, either press `Enter`,
+or click away.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:104]:
```suggestion
number, select that figure, and type in the desired number in its place. Then, either press
:kbd:`Enter`, or click away.
```

File: content/applications/marketing/surveys/create.rst [hunk 9]
```diff
@@ -1,224 +1,331 @@
...
-require a login to participate in the survey. If this option is activated, an :guilabel:`Attempts
-Limit` field also populates, in which the number of survey attempts is defined for the participant.
+   If that complete :guilabel:`Session Link` (ending in the custom :guilabel:`Session Code`) is used
+   by participants to access the live session survey, that link would *already* be entered for them.
+   At that point, they'd simply have to wait until the host of the live session begins the survey,
+   and then they'd be able to enter.
 
-Live Session
-------------
+   If the :guilabel:`Session Link` (ending in the custom :guilabel:`Session Code`)  is sent
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:240]:
```suggestion
   If the :guilabel:`Session Link` (ending in the custom :guilabel:`Session Code`) is sent
```

File: content/applications/marketing/surveys/create.rst [hunk 10]
```diff
@@ -1,224 +1,331 @@
...
+
+.. image:: create/survey-form-buttons.png
+   :align: center
+   :alt: The various buttons on a survey form in Odoo Surveys.
+
+Those buttons are the following:
+
+- :guilabel:`Share`: click to reveal a :guilabel:`Share a Survey` pop-up form that can be used to
+  invite potential participants to the survey — complete with a :guilabel:`Survey Link` that can be
+  copied and sent to potential participants, and a :guilabel:`Send by Email` toggle switch.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:280]:
```suggestion
- :guilabel:`Share`: click to reveal a :guilabel:`Share a Survey` modal that can be used to invite
  potential participants to the survey — complete with a :guilabel:`Survey Link` that can be copied
  and sent to potential participants, and a :guilabel:`Send by Email` toggle switch.
```

File: content/applications/marketing/surveys/create.rst [hunk 11]
```diff
@@ -1,224 +1,331 @@
...
+
+   .. image:: create/share-survey-popup.png
+      :align: center
+      :alt: The 'Share a Survey' pop-up window that appears in the Odoo Surveys application.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:284]: Align image with bullet point text
```suggestion
  .. image:: create/share-survey-popup.png
     :align: center
     :alt: The 'Share a Survey' pop-up window that appears in the Odoo Surveys application.
```

File: content/applications/marketing/surveys/create.rst [hunk 12]
```diff
@@ -1,224 +1,331 @@
...
+
+  When the :guilabel:`Send by Email` toggle is in the 'on' position, indicated by a green switch,
+  additional fields appear, in which :guilabel:`Recipients` and a :guilabel:`Subject` can be added
+  to the email. Below that, a dynamic email template, complete with a :guilabel:`Start
+  Certification` button appears, which can also be modified.
+
+   .. image:: create/share-survey-popup-email-toggle.png
+      :align: center
+      :alt: The 'Share a Survey' pop-up window in Odoo Surveys with the Send by Email toggled on.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:293]:
```suggestion
  .. image:: create/share-survey-popup-email-toggle.png
     :align: center
     :alt: The 'Share a Survey' pop-up window in Odoo Surveys with the Send by Email toggled on.
```

File: content/applications/marketing/surveys/create.rst [hunk 13]
```diff
@@ -1,224 +1,331 @@
...
+
+  Once modifications are complete, click :guilabel:`Send` to send that email invite to all the email
+  addresses/contacts listed in the :guilabel:`Recepients` field.
+
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:297]: No need for an extra line between bullet points of the same level
```suggestion
```

File: content/applications/marketing/surveys/create.rst [hunk 14]
```diff
@@ -1,224 +1,331 @@
...
+- :guilabel:`See results`: this button **only** appears if there has been at least one participant
+  who has compelted the survey. Clicking :guilabel:`See results` reveals a separate tab containing
+  a visual analysis of the survey questions and resposnes. For more information, check out the
+  documentation we have on that.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:302]: Typo and reference doc, and remove extra line
```suggestion
  who has completed the survey. Clicking :guilabel:`See results` reveals a separate tab containing
  a visual analysis of the survey questions and responses. For more information, check out the
  :doc:`scoring surveys <scoring>` documentation.
```

File: content/applications/marketing/surveys/create.rst [hunk 15]
```diff
@@ -1,224 +1,331 @@
...
-=========================
+  Additionally, when :guilabel:`Create Live Session` has been clicked, and the *Session Manager* tab
+  has been opened, the :guilabel:`Create Live Session` button on the survey form is replaced with
+  two new buttons: :guilabel:`Open Session Manager` and :guilabel:`Close Live Session`.
 
-Once the survey is created and saved, run a test to check for possible errors before finally
-sending it out to the participants by clicking :guilabel:`Test` in the upper left corner of the
-survey template page.
+  Clicking :guilabel:`Open Session Manager` opens another separate tab to the *Session Manager*, and
+  clicking :guilabel:`Close Live Session` closes, and subsequently ends, the live session.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:314]:
```suggestion
```

File: content/applications/marketing/surveys/create.rst [hunk 16]
```diff
@@ -1,224 +1,331 @@
...
+- :guilabel:`Test`: clicking this button opens a new tab to a test version of the survey, in order
+  for the user to check for errors or inconsistencies, from the point-of-view of a participant.
+  Users can tell if they are in a test version of the survey if there is a blue banner at the top of
+  the screen, reading: `This is a Test Survey --> Edit Survey`.
 
-To return to the survey template form in the backend, simply click the :guilabel:`This is a test
-survey. Edit Survey` link in the blue banner along the top of the page. Once Odoo redirects the
-page to the survey template in the backend, make any further changes, as needed, before officially
-sending the survey out to participants.
+  If the link in the blue banner is clicked, Odoo returns the user to the survey form.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:321]:
```suggestion
```

File: content/applications/marketing/surveys/create.rst [hunk 17]
```diff
@@ -1,224 +1,331 @@
...
+- :guilabel:`Close`: clicking this button closes the survey (i.e. archives it), which is represented
+  by a red :guilabel:`Archived` banner across the top-right corner of the survey form.
 
-As answers are received, check them by clicking the :guilabel:`Answers` smart button on the survey
-template form, or the :guilabel:`See Results` button in the upper left corner. To end the survey,
-click the :guilabel:`Close` button on the survey template form.
+  When this button is clicked, and the survey is closed, a single button appears in the upper-right
+  corner of the survey form, titled: :guilabel:`Reopen`. When :guilabel:`Reopen` is clicked the
+  survey is reopened (i.e. unarchived), and the :guilabel:`Archived` banner is removed from the
+  survey form.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:331]: Suggestion to link to other docs
```suggestion
  survey form.
  
.. seealso::
   - :doc:`time_random`
   - :doc:`scoring`
   
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @ksc-odoo, nice job on the improvements in this doc! I have some technical corrections and suggestions, please see below.

Just a note for consideration, once https://github.com/odoo/documentation/pull/8381 is merged, it can be referenced in this documentation. In doing so, would the **Live Session section** be redundant?

I also would like to mention that this scope _could_ use a restructuring where this `create.rst` general overview content is placed directly in `surveys.rst`. As well, `time_random.rst` can be renamed to `questions.rst` per https://github.com/odoo/documentation/pull/8259. This all can be handled in a new [MOV] pr, but figured I would mention it here.

Nice job, tag me for another quick look once these are addressed, thank you! :)

File: content/applications/marketing/surveys/create.rst (hunk 4 above)
Comment (ksc-odoo) [reply content/applications/marketing/surveys/create.rst:74]: I know this comment was *before* we decided to stick with "pop-up..." for the novice readers...so, I'm going to disregard. Appreciate the suggestion, though 👍

File: content/applications/marketing/surveys/create.rst (hunk 6 above)
Comment (ksc-odoo) [reply content/applications/marketing/surveys/create.rst:81]: implementing everything *but* the "modal" suggestion 👍

**Comment by ksc-odoo** [comment]
Hey @samueljlieber -- Thanks for all the helpful feedback. I implemented all your suggestions, and now this one's ready for another look from you, whenever you get a chance.

Also, like we discussed last week, I believe this PR, too -- because it's basically a complete rewrite of the previous doc (i.e. most, if not all, content is brand new) -- and think this one is definitely worthy of a 5-point designation.

File: content/applications/marketing/surveys/create.rst [hunk 18]
```diff
@@ -1,224 +1,333 @@
...
+  additional fields appear, in which :guilabel:`Recipients` and a :guilabel:`Subject` can be added
+  to the email. Below that, a dynamic email template, complete with a :guilabel:`Start
+  Certification` button appears, which can also be modified.
+
+  .. image:: create/share-survey-popup-email-toggle.png
+     :align: center
+     :alt: The 'Share a Survey' pop-up window in Odoo Surveys with the Send by Email toggled on.
+
+  Once modifications are complete, click :guilabel:`Send` to send that email invite to all the email
+  addresses/contacts listed in the :guilabel:`Recepients` field.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:297]: Field name typo
```suggestion
  addresses/contacts listed in the :guilabel:`Recipients` field.
```

File: content/applications/marketing/surveys/create.rst [hunk 19]
```diff
@@ -1,224 +1,333 @@
...
+- :guilabel:`See results`: this button **only** appears if there has been at least one participant
+  who has completed the survey. Clicking :guilabel:`See results` reveals a separate tab containing
+  a visual analysis of the survey questions and responses. For more information, check out the
+  :doc:`scoring surveys <scoring>` documentation.
+
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:302]: No need for an extra line between bullet points of the same level
```suggestion
```

File: content/applications/marketing/surveys/create.rst [hunk 20]
```diff
@@ -1,224 +1,333 @@
...
+  clicking :guilabel:`Close Live Session` closes, and subsequently ends, the live session.
+- :guilabel:`Test`: clicking this button opens a new tab to a test version of the survey, in order
+  for the user to check for errors or inconsistencies, from the point-of-view of a participant.
+  Users can tell if they are in a test version of the survey if there is a blue banner at the top of
+  the screen, reading: `This is a Test Survey --> Edit Survey`.
+
+  If the link in the blue banner is clicked, Odoo returns the user to the survey form.
+- :guilabel:`Print`: clicking this button opens a new tab to a printable version of the survey that
+  the user can proceed to print for their records.
+
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/create.rst:322]:
```suggestion
```

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo, thank you for addressing my suggestions! This PR looks great! I caught a couple more small fixes, be sure to run `make review` as there were a few early line breaks as well. I am approving now and you can merge at your discretion, thank you!
.....
@robodoo delegate=ksc-odoo

**Comment by ksc-odoo** [comment]
@robodoo r+

**Pull request closed without merging**